app-misc/hello/hello-2.10.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="GNU Hello program"
+DESCRIPTION="The GNU Hello program produces a familiar, friendly greeting.\
+Yes, this is another implementation of the classic program that prints \"Hello, world!\" when you run it.\
+However, unlike the minimal version often seen, GNU Hello processes its argument list to modify its behavior,\
+supports greetings in many languages, and so on.\
+The primary purpose of GNU Hello is to demonstrate how to write other programs that do these things;\
+it serves as a model for GNU coding standards and GNU maintainer practices."
+HOMEPAGE="https://www.gnu.org/software/hello/"
+COPYRIGHT="1992-2014 Free Software Foundation, Inc."
+LICENSE="GNU GPL v3"
+REVISION="2"
+SOURCE_URI="http://ftp.gnu.org/gnu/hello/hello-$portVersion.tar.gz"
+CHECKSUM_SHA256="31e066137a962676e89f69d1b65382de95a7ef7d914b8cb956f41ea72e0f516b"
+
+ARCHITECTURES="x86 x86_gcc2 x86_64"
+
+PROVIDES="
+	hello = $portVersion
+	cmd:hello = $portVersion
+	"
+REQUIRES="
+	haiku
+	gnulib
+	lib:libintl
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libgettextlib >= 0.18.1
+	
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:gcc
+	cmd:make
```

Comment:
* No need for the empty line after `devel:libgettextlib`
* Sorting `BUILD_PREREQUIRES`

```diff
 	devel:libgettextlib >= 0.18.1
-	
 	"
 BUILD_PREREQUIRES="
 	cmd:autoconf
 	cmd:gcc
-	cmd:make
-	cmd:help2man
 	cmd:gettext
+	cmd:help2man
+	cmd:make
 	"
```